src/views/product-root-docs-path-landing/components/marketing-content/index.tsx

```diff
@@ -21,8 +21,9 @@ const ProductRootDocsPathLandingMarketingContent = ({ blocks }) => {
 
 	return (
 		<div className={s.root}>
-			{blocks.map((block, index) => {
+			{blocks.map((block, index: number) => {
 				if (block.type === 'paragraph') {
+					// eslint-disable-next-line react/no-array-index-key
 					return <LandingPageBlocks.ParagraphBlock {...block} key={index} />
```

Comment:
What about using `key={block.title}` here like we do below? Or does block not have a title attribute here?

Reply:
Awesome, thanks for that context!